src/posit/connect/client.py

```diff
@@ -35,14 +35,23 @@ def __init__(
         # Store the Session object.
         self.session = session
 
-        # Place to cache the server settings
-        self.server_settings = None
+        # Internal attributes to hold settings we fetch lazily
+        self._server_settings = None
+        self._current_user: Optional[User] = None
```

Comment:
Since we are moving away from caching in `Users`, should we remove it here for consistency?

Reply:
Saving the GUID seems safe, but other information regarding the `current_user` could change between calls to `client.me`.

This morning, at least, I like the idea of not caching anything by convention. We can add it back in if the server request load becomes burdensome.

Reply:
Yep, I agree. I'll remove, then merge.